autonomi/src/client/registers.rs

```diff
@@ -333,10 +341,11 @@ impl Client {
             expected_holders: Default::default(),
             is_register: true,
         };
+
         let put_cfg = PutRecordCfg {
             put_quorum: Quorum::All,
             retry_strategy: None,
-            use_put_record_to: Some(vec![payee]),
+            use_put_record_to: Some(payees), // CODE REVIEW: should we put to all or just one here?
```

Comment:
All sounds good. Maybe in the future, to strain the network less we could put to 1 by 1 until the data is stored

Reply:
This can come as an optimization